src/navigation/nav.story.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { MOCK_MEMBER } from 'meetup-web-mocks/lib/api';
+import { MOCK_NOTIFICATIONS_LIST } from 'meetup-web-mocks/lib/notifications/api';
+
+import { decorateWithBasics } from '../utils/decorators';
+
+import Nav from './Nav';
+
+const navItems = {
+	updatesLabel: 'Updates',
+	login: {
+		link: 'meetup.com/login',
+		label: 'Login',
+	},
+	signup: {
+		link: 'meetup.com/signup',
+		label: 'Signup',
+	},
+	proDashboard: {
+		link: 'meetup.com/pro',
+		label: 'Pro Dashboard',
+		mobileLabel: 'Dashboard',
+		proLogo: MOCK_MEMBER.photo.photo_link,
+	},
+	explore: {
+		link: 'meetup.com/explore',
```

Comment:
nitpick - i know this is for the storybook story, but is `/explore` the correct url for the Explore button? on the current nav, we direct to `/find/events`